### DISCUSSION TITLE
Is this capable of making a very simple game language?

### DISCUSSION THREAD

**Discussion by photonstorm** [peggyjs/peggy#419] [Q&A]
Before I spend ages digging in to this I just wondered if this is just the wrong sort of use for Peggy?

I'd like to make a simple language similar to AMOS from the Amiga, so focused on a few commands that do graphics and move things around.

Obviously I can use canvas to handle to rendering but would Peggy even cope with a language where you make bits of code that repeat continuously in a loop? For example moving a sprite across a "screen".

To be clear I want to make a really simple and small basic style language, but looping and maintaining state (outside of the user code) is important.

Am I barking up the wrong tree here?

**Comment by hildjj** [comment]
Should work fine.  Join us on Discord in the [#grammar-help](https://discord.gg/3uxhnvRM) channel to get tips and tricks for getting started.
